### DISCUSSION TITLE
感觉banner的图片可以给一个设置展示位置的（center | top | bottom 之类的)

### DISCUSSION THREAD

**Discussion by Cxpller** [MaLuns/hexo-theme-async#19] [Q&A 疑难杂症] [answered]

**Comment by Cxpller** [comment]
![image](https://user-images.githubusercontent.com/76519622/194975632-7c04fba4-2df7-4741-a0c3-44588b5ae80d.png)

**Comment by MaLuns** [comment]
这个可以有，下个版本加上

**Comment by MaLuns** [comment, marked as answer]
1.2.2 加上了😀

**Reply by Cxpller** [reply]
😊😊大大nb捏，如果我要从1.2.1更新到1.2.2我需要做什么不，还是说要重新构建

**Reply by MaLuns** [reply]
> 😊😊大大nb捏，如果我要从1.2.1更新到1.2.2我需要做什么不，还是说要重新构建

如果你是 npm 安装的，直接升级包就行了
